File: lua/refactoring/tests/refactor/123/lua/example/inline_var.start.lua
-- stylua: ignore start

local function orderCalculation(order)
    local basePrice = order.quantity * order.itemPrice
    return basePrice
        - math.max(0, order.quantity - 500) * order.itemPrice * 0.05
        + math.min(basePrice * 0.1, 100)
end

File: lua/refactoring/tests/refactor/123/java/example/inline_var.start.java
class Order {
  int quantity;
  int itemPrice;
}

class Inline {
  double orderCalculation(Order order) {
    float basePrice = order.quantity * order.itemPrice;
    return basePrice -
        Math.max(0, order.quantity - 500) * order.itemPrice * 0.05 +
        Math.min(basePrice * 0.1, 100.0);
  }
}
